src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -2,6 +2,6 @@
 
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[(int) (Math.random() * Colors.values().length)].toString();
```

Comment:
It would be cool to move some logic out of this line to increase the readability. 
Remember about private methods ;)